common/src/main/java/com/gregtechceu/gtceu/integration/emi/oreprocessing/GTOreProcessingEmiCategory.java

```diff
@@ -0,0 +1,60 @@
+package com.gregtechceu.gtceu.integration.emi.oreprocessing;
+
+import com.gregtechceu.gtceu.GTCEu;
+import com.gregtechceu.gtceu.api.data.chemical.material.Material;
+import com.gregtechceu.gtceu.api.machine.MachineDefinition;
+import com.gregtechceu.gtceu.api.recipe.GTRecipeType;
+import com.gregtechceu.gtceu.api.registry.GTRegistries;
+import com.gregtechceu.gtceu.integration.rei.oreprocessing.GTOreProcessingDisplayCategory;
+import com.lowdragmc.lowdraglib.emi.ModularUIEmiRecipeCategory;
+import dev.emi.emi.api.EmiRegistry;
+import dev.emi.emi.api.stack.EmiStack;
+import me.shedaniel.rei.api.common.util.EntryStacks;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.Items;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.gregtechceu.gtceu.api.data.chemical.material.properties.PropertyKey.ORE;
+import static com.gregtechceu.gtceu.common.data.GTRecipeTypes.*;
+
+public class GTOreProcessingEmiCategory extends ModularUIEmiRecipeCategory {
+    public static final GTOreProcessingEmiCategory CATEGORY = new GTOreProcessingEmiCategory();
+    public GTOreProcessingEmiCategory() {
+        super(new ResourceLocation(GTCEu.MOD_ID + ":ore_processing_diagram"), EmiStack.of(Items.IRON_ORE));
```

Comment:
wtf is this? just do `GTCEu.id("ore_processing_diagram")`

Reply:
I think it's from the file I copied

Reply:
No, not quite
Remove the `new ResourceLocation(`